src/lib/deploy.js

```diff
@@ -519,7 +499,6 @@ function getAccountQuery(publicKey) {
   return `
   query {
     account(publicKey: "${publicKey}") {
-      publicKey
```

Comment:
Yeah, this was intentional. My reasoning for first adding `publicKey` in this query was to see if it would be useful anywhere later in the cli, but it hasn't turned out to be the case so I'm just going to remove it.

Reply:
cool